app/build.gradle

```diff
@@ -66,6 +66,7 @@ android {
 
             buildConfigField "String", "IP_SIMPLE_API_ACCESS_KEY", "\"$local_properties.ip_simple_api_access_key\""
             buildConfigField "String", "ANDROID_SIMPLE_API_ACCESS_KEY", "\"$local_properties.android_simple_api_access_key\""
+            buildConfigField "String", "FIREBASE_WEB_API_KEY", "\"$local_properties.firebase_web_api_key\""
```

Comment:
Are you sure that you need a new api key? I think `IP_SIMPLE_API_ACCESS_KEY` would work. If you don't have that key, simple create one in Google api console. If you're not sure, feel free to ask us at https://gitter.im/gdg-x/frisbee

Reply:
this way may not right. hard to say